cli/internal/daemonclient/daemonclient.go

```diff
@@ -33,9 +34,14 @@ func New(client *connector.Client) *DaemonClient {
 
 // GetChangedOutputs implements runcache.OutputWatcher.GetChangedOutputs
 func (d *DaemonClient) GetChangedOutputs(ctx context.Context, hash string, repoRelativeOutputGlobs []string) ([]string, int, error) {
```

Comment:
Are the output globs being passed in using a `\` path separator on Windows?

Reply:
some parts of the glob have `\` separators, I think from filepaths that we concat with the globs.